src/qs_scf.F

```diff
@@ -598,11 +598,12 @@ SUBROUTINE scf_env_do_scf(scf_env, scf_control, qs_env, converged, should_stop,
          dft_control%qs_control%cdft_control%total_steps = &
          dft_control%qs_control%cdft_control%total_steps + total_steps
 
-      IF (.NOT. converged) CPWARN("SCF run NOT converged")
```

Comment:
Maybe we should keep the old warning message? I'd expect it's hard coded in many places.